src/pages/home/report/ReportActionItem.js

```diff
@@ -66,11 +77,12 @@ class ReportActionItem extends Component {
     shouldComponentUpdate(nextProps, nextState) {
         return this.state.isPopoverVisible !== nextState.isPopoverVisible
             || this.props.displayAsGroup !== nextProps.displayAsGroup
+            || this.props.draftMessage !== nextProps.draftMessage
             || this.props.isMostRecentIOUReportAction !== nextProps.isMostRecentIOUReportAction
             || this.props.hasOutstandingIOU !== nextProps.hasOutstandingIOU
             || this.props.iouReportID !== nextProps.iouReportID
-            || (this.props.shouldDisplayNewIndicator !== nextProps.shouldDisplayNewIndicator)
-            || !_.isEqual(this.props.action, nextProps.action);
+            || !_.isEqual(this.props.action, nextProps.action
```

Comment:
NAB but why switch these? Only asking because it doesn't seem necessary and `_.isEqual` is a (negligibly) slower comparison than `!==`.

Reply:
Bad conflict merge!